projects/ngx-smart-banner/src/lib/ngx-smart-banner.component.html

```diff
@@ -16,6 +22,43 @@
             <div class="smartbanner__info--author">
                 {{ settings.author }}
             </div>
+            <div class="smartbanner__info--rating">
+                <div *ngIf="reviewAverage === 1">
```

Comment:
This is possible using an *ngFor using the star icon i mentioned above it would be:

```
<app-star *ngFor="let in of [0,1,2,3,4];let i = index" [solid]="reviewAverage <= index"></app-star>
```